demos/common/models/include/models/segmentation_model.h

```diff
@@ -21,8 +21,9 @@
 class SegmentationModel : public ModelBase {
 public:
     /// Constructor
-    /// @param model_nameFileName of model to load
-    SegmentationModel(const std::string& modelFileName) : ModelBase(modelFileName) {}
+    /// @param modelFileName name of model to load
+    /// @param useAutoResize - if true, image will be resized by IE.
```

Comment:
```suggestion
    /// @param useAutoResize - if true, image will be resized by IE.
    /// Otherwise, image will be preprocessed and resized using OpenCV routines.
```